### PR TITLE
Update docker-library images

### PR DESCRIPTION
**Pull request by tianon**
- `buildpack-deps`: remove utopic (EOL)
- `celery`: no functional change (`.travis.yml` added)
- `golang`: 1.5rc1 (docker-library/golang#56)
- `hello-world`: https (docker-library/hello-world#5), build-in-container (docker-library/hello-world#6)
- `java`: 8u66
- `mariadb`: 10.0.21+maria-1~jessie and 5.5.45+maria-1~wheezy

### CODE REVIEW

**Comment by tianon** [comment]
cc @thajeztah (here's the `hello-world` fix :innocent:)

**Comment by thaJeztah** [comment]
Woohoo! LGTM

image

File: library/hello-world [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# maintainer: Tianon Gravi <admwiggin@gmail.com> (@tianon)
+# maintainer: InfoSiftr <github@infosiftr.com> (@infosiftr)
```
Comment (thaJeztah) [inline library/hello-world]: Aww, you got demoted?

Comment (tianon) [inline library/hello-world]: Lol naw, just properly reflecting reality. :+1:

**Comment by tianon** [comment]
Build test of #956; 55d19f4ec35a598265155a37ed09354dbe568efd (`buildpack-deps`, `celery`, `golang`, `hello-world`, `java`, `mariadb`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/55d19f4ec35a598265155a37ed09354dbe568efd/library/buildpack-deps"
$ bashbrew build "$url"
Processing buildpack-deps:jessie-curl ...
Processing buildpack-deps:curl ...
Processing buildpack-deps:jessie-scm ...
Processing buildpack-deps:scm ...
Processing buildpack-deps:jessie ...
Processing buildpack-deps:latest ...
Processing buildpack-deps:precise-curl ...
Processing buildpack-deps:precise-scm ...
Processing buildpack-deps:precise ...
Processing buildpack-deps:sid-curl ...
Processing buildpack-deps:sid-scm ...
Processing buildpack-deps:sid ...
Processing buildpack-deps:stretch-curl ...
Processing buildpack-deps:stretch-scm ...
Processing buildpack-deps:stretch ...
Processing buildpack-deps:trusty-curl ...
Processing buildpack-deps:trusty-scm ...
Processing buildpack-deps:trusty ...
Processing buildpack-deps:vivid-curl ...
Processing buildpack-deps:vivid-scm ...
Processing buildpack-deps:vivid ...
Processing buildpack-deps:wheezy-curl ...
Processing buildpack-deps:wheezy-scm ...
Processing buildpack-deps:wheezy ...
Processing buildpack-deps:wily-curl ...
Processing buildpack-deps:wily-scm ...
Processing buildpack-deps:wily ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing buildpack-deps:jessie-curl
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:jessie-scm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:jessie
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:precise-curl
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:precise-scm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:precise
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:sid-curl
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:sid-scm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:sid
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:stretch-curl
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:stretch-scm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:stretch
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:trusty-curl
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:trusty-scm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:trusty
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:vivid-curl
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:vivid-scm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:vivid
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:wheezy-curl
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:wheezy-scm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:wheezy
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:wily-curl
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:wily-scm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing buildpack-deps:wily
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/55d19f4ec35a598265155a37ed09354dbe568efd/library/celery"
$ bashbrew build "$url"
Fetching celery (git://github.com/docker-library/celery) ...
Processing celery:3.1.18 ...
Processing celery:3.1 ...
Processing celery:3 ...
Processing celery:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing celery:3.1.18
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/55d19f4ec35a598265155a37ed09354dbe568efd/library/golang"
$ bashbrew build "$url"
Fetching golang (git://github.com/docker-library/golang) ...
Processing golang:1.3.3 ...
Processing golang:1.3 ...
Processing golang:1.3.3-onbuild ...
Processing golang:1.3-onbuild ...
Processing golang:1.3.3-cross ...
Processing golang:1.3-cross ...
Processing golang:1.3.3-wheezy ...
Processing golang:1.3-wheezy ...
Processing golang:1.4.2 ...
Processing golang:1.4 ...
Processing golang:1 ...
Processing golang:latest ...
Processing golang:1.4.2-onbuild ...
Processing golang:1.4-onbuild ...
Processing golang:1-onbuild ...
Processing golang:onbuild ...
Processing golang:1.4.2-cross ...
Processing golang:1.4-cross ...
Processing golang:1-cross ...
Processing golang:cross ...
Processing golang:1.4.2-wheezy ...
Processing golang:1.4-wheezy ...
Processing golang:1-wheezy ...
Processing golang:wheezy ...
Processing golang:1.5rc1 ...
Processing golang:1.5 ...
Processing golang:1.5rc1-onbuild ...
Processing golang:1.5-onbuild ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing golang:1.3.3
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.3.3-onbuild
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.3.3-cross
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.3.3-wheezy
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.4.2
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.4.2-onbuild
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.4.2-cross
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.4.2-wheezy
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.5rc1
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing golang:1.5rc1-onbuild
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/55d19f4ec35a598265155a37ed09354dbe568efd/library/hello-world"
$ bashbrew build "$url"
Cloning hello-world (git://github.com/docker-library/hello-world) ...
Processing hello-world:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing hello-world:latest
	'cve-2014--shellshock' [1/1]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/55d19f4ec35a598265155a37ed09354dbe568efd/library/java"
$ bashbrew build "$url"
Fetching java (git://github.com/docker-library/java) ...
Processing java:openjdk-6b35-jdk ...
Processing java:openjdk-6b35 ...
Processing java:openjdk-6-jdk ...
Processing java:openjdk-6 ...
Processing java:6b35-jdk ...
Processing java:6b35 ...
Processing java:6-jdk ...
Processing java:6 ...
Processing java:openjdk-6b35-jre ...
Processing java:openjdk-6-jre ...
Processing java:6b35-jre ...
Processing java:6-jre ...
Processing java:openjdk-7u79-jdk ...
Processing java:openjdk-7u79 ...
Processing java:openjdk-7-jdk ...
Processing java:openjdk-7 ...
Processing java:7u79-jdk ...
Processing java:7u79 ...
Processing java:7-jdk ...
Processing java:7 ...
Processing java:openjdk-7u79-jre ...
Processing java:openjdk-7-jre ...
Processing java:7u79-jre ...
Processing java:7-jre ...
Processing java:openjdk-8u66-jdk ...
Processing java:openjdk-8u66 ...
Processing java:openjdk-8-jdk ...
Processing java:openjdk-8 ...
Processing java:8u66-jdk ...
Processing java:8u66 ...
Processing java:8-jdk ...
Processing java:8 ...
Processing java:jdk ...
Processing java:latest ...
Processing java:openjdk-8u66-jre ...
Processing java:openjdk-8-jre ...
Processing java:8u66-jre ...
Processing java:8-jre ...
Processing java:jre ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing java:openjdk-6b35-jdk
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing java:openjdk-6b35-jre
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing java:openjdk-7u79-jdk
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing java:openjdk-7u79-jre
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing java:openjdk-8u66-jdk
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing java:openjdk-8u66-jre
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/55d19f4ec35a598265155a37ed09354dbe568efd/library/mariadb"
$ bashbrew build "$url"
Fetching mariadb (git://github.com/docker-library/mariadb) ...
Processing mariadb:10.0.21 ...
Processing mariadb:10.0 ...
Processing mariadb:10 ...
Processing mariadb:latest ...
Processing mariadb:5.5.45 ...
Processing mariadb:5.5 ...
Processing mariadb:5 ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing mariadb:10.0.21
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'mysql-basics' [4/5]...passed
	'mysql-initdb' [5/5]...ERROR 1146 (42S02) at line 1: Table 'my cool mysql database.test' doesn't exist
failed
testing mariadb:5.5.45
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'mysql-basics' [4/5]...passed
	'mysql-initdb' [5/5]...ERROR 1146 (42S02) at line 1: Table 'my cool mysql database.test' doesn't exist
failed
```

There is at least one failure in the above build log.

**Comment by tianon** [comment]
(The `mysql-initdb` failures are expected for now.)

**Comment by yosifkit** [comment]
LGTM

**Comment by thaJeztah** [comment]
All fresh and shiny

image

**Pull request merged**